### PR TITLE
ffmpeg: build -dbg package properly

### LINKED ISSUE #28931: ffmpeg-dbg package doesn't contain debug symbols
### System

* xuname: Void 5.10.16_1 x86_64 AuthenticAMD notuptodate rrrmdDFFFFFF
* package: ffmpeg-dbg-4.3.1_6

### Expected behavior
FFmpeg binaries (ffmpeg, ffplay, ffprobe) from the ffmpeg-dbg package should contain debug symbols.

### Actual behavior
The binaries from the ffmpeg-dbg package are already stripped. The build process of FFmpeg creates a stripped binary and one with debug symbols with a `_g` suffix. See https://ffmpeg.org/faq.html#Why-are-the-ffmpeg-programs-devoid-of-debugging-symbols_003f for more details.

### Steps to reproduce the behavior
1. Install ffmpeg-dbg
2. Run `gdb ffmpeg` which declares no debug symbols were found

### PR DESCRIPTION
**Pull request by sgn**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

Close #28931

### CODE REVIEW

File: srcpkgs/ffmpeg/template [hunk 1]
```diff
@@ -99,7 +100,6 @@ do_configure() {
 		$(vopt_if webp '--enable-libwebp')
 }
 do_build() {
-	rm ${XBPS_WRAPPERDIR}/strip
```
Comment (ericonr) [inline srcpkgs/ffmpeg/template:102]: This was added in a0d88ba8496402b76ab503a59b3990e441f878ab for unknown reasons >.<

**Review by ericonr — COMMENTED** [review]
```
STRIP   libswscale/x86/yuv_2_rgb.o
skipping strip -x libswscale/x86/yuv_2_rgb.o
```

so `--disable-stripping` isn't enough, but our wrapper fixes it, I guess.

**Comment by ericonr** [comment]
Tested with debuginfod, this seems to have fixed it. The i686 failure is due to old `dolphin-emu`, which should probably be cleaned up.

**Comment by sgn** [comment]
`--disable-stripping` is enough. That echo come from their Makefile.

File: srcpkgs/ffmpeg/template (hunk 1 above)
Comment (sgn) [reply srcpkgs/ffmpeg/template:102]: W/o that line and `--disable-stripping`, the build will run into failure.

**Pull request merged**